op-batcher/batcher/espresso.go

```diff
@@ -246,6 +291,30 @@ const VERIFY_RECEIPT_TIMEOUT = 4 * time.Second
 // retrying a job that failed to verify the receipt.
 const VERIFY_RECEIPT_RETRY_DELAY = 100 * time.Millisecond
 
+// Evaluate the verification job.
+func evaluateVerification(jobResp espressoVerifyReceiptJobResponse) JobEvaluation {
+	err := jobResp.err
+
+	// If there's no error, continue handling the verification.
+	if err == nil {
+		return Handle
+	}
+
+	// If the hash is invalid, skip the verification.
+	if strings.Contains(err.Error(), "hash is nil") {
+		log.Warn("Hash is nil, skipping")
+		return Skip
+	}
+
+	// If the verifcation times out, degrade to the submission phase and try again.
+	if have := time.Now(); have.Sub(jobResp.job.start) > VERIFY_RECEIPT_TIMEOUT {
+		return RetrySubmission
```

Comment:
I guess this should be `RetryVerification`, but as mentioned [above](https://github.com/EspressoSystems/optimism-espresso-integration/pull/161/files#r2112939127) I would prefer to just use `Retry`.